tests/data/config.json

```diff
@@ -28,8 +28,8 @@
                         "dbName": "name",
                         "type": "varchar",
                         "size": 255,
-                        "nullable": null,
-                        "default": null
+                        "nullable": false,
```

Comment:
Mozno by bolo dobre testy neupravovat ale kopirovat ... tj. mat test na to, ked tie hodnoty nie su v configu, su null, a su vyplnene (aby sa tam nezaniesla nejaka nova chyba).